.changeset/rich-apes-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hono/vite-dev-server': patch
```

Comment:
You should replace `patch` with `minor` since this is a new feature.

Reply:
Done! Thanks!